webrtc.html

```diff
@@ -5851,12 +5852,15 @@ <h2>Methods</h2>
                   <p>Run the following steps in parallel:</p>
                   <ol>
                     <li>
-                      <p>Determine if negotiation is needed to transmit
-                      <var>withTrack</var> in place of the sender's existing
-                      track. Negotiation is not needed if <var>withTrack</var>
+                      <p>Determine if negotiation would be needed in order to 
+                      replace <var>sender</var>'s existing track with
+                      <var>withTrack</var>.
+                      Negotiation is not needed if <var>withTrack</var>
                       is null. Ignore which <code>MediaStream</code> the track
                       resides in and the <code>id</code> attribute of the track
-                      in this determination. If negotiation is needed, then
+                      in this determination. Also, ignore
+                      <var>transceiver</var>'s <a>[[\Direction]]</a> slot in
```

Comment:
More broadly, I would say *"ignore whether the sender already needs negotiation, in this determination"*. The goal of the determination is to see whether the two tracks are compatible.

Reply:
> The goal of the determination is to see whether the two tracks are compatible

Yes, or rather if the new track is compatible to be transmitted without having to alter any "media" part of the SDP (with "media" I mean codec, sampling rate, and such).

The hard part it to put that in words...
> "ignore whether the sender already needs negotiation, in this determination"

I'm not sure that helps.

Reply:
I think I now understand, you want to add "ignore whether the sender already needs negotiation, in this determination" in addition to "ignore MediaStream stuff" and "ignore Direction stuff". With that I agree, I'll update the PR.